### PR TITLE
Migrate off Infura to Alchemy.

### PR DESCRIPTION
**Pull request by jnaviask**
## Description

Switches all infura connections to alchemy across the app.

## Motivation and Context

First step to achieving arbitrary ETH chain support.

## How has this been tested?

Ran locally and verified logins and data loading. Needs QA!

## Have proper tags been added (for bug, enhancement, breaking change)?
- [x] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [x] yes, but I did not run tests
- [ ] no

### CODE REVIEW

**Review by timolegros — COMMENTED** [review]
Wouldn't it be more future-proof if we had some enum in `shared/types.ts` so that we could change the URLs/token if needed?
```
export enum AlchemyUrls {
  httpMainnet = ''https://eth-mainnet.alchemyapi.io/v2/cNC4XfxR7biwO2bfIO5aKcs9EMPxTQfr",
  httpRopsten = "https://eth-ropsten.alchemyapi.io/v2/2xXT2xx5AvA3GFTev3j_nB9LzWdmxPk7",
  wssMainnet = "wss://eth-mainnet.alchemyapi.io/v2/cNC4XfxR7biwO2bfIO5aKcs9EMPxTQfr",
  wssRopsten = "wss://eth-ropsten.alchemyapi.io/v2/2xXT2xx5AvA3GFTev3j_nB9LzWdmxPk7"
}
```

**Comment by jnaviask** [comment]
> Wouldn't it be more future-proof if we had some enum in `shared/types.ts` so that we could change the URLs/token if needed?
> 
> ```
> export enum AlchemyUrls {
>   httpMainnet = ''https://eth-mainnet.alchemyapi.io/v2/cNC4XfxR7biwO2bfIO5aKcs9EMPxTQfr",
>   httpRopsten = "https://eth-ropsten.alchemyapi.io/v2/2xXT2xx5AvA3GFTev3j_nB9LzWdmxPk7",
>   wssMainnet = "wss://eth-mainnet.alchemyapi.io/v2/cNC4XfxR7biwO2bfIO5aKcs9EMPxTQfr",
>   wssRopsten = "wss://eth-ropsten.alchemyapi.io/v2/2xXT2xx5AvA3GFTev3j_nB9LzWdmxPk7"
> }
> ```

The next step in this PR is to remove hardcoded APIs like these altogether, so don't worry about it.

**Review by timolegros — APPROVED** [review]

**Review by zakhap — APPROVED** [review]
Great job. Didn't seem to break any the pages I QA'd or the event fetching.

File: client/scripts/views/modals/create_community_modal.ts [hunk 1]
```diff
@@ -753,7 +753,7 @@ const ERC20Form: m.Component<ERC20FormAttrs, ERC20FormState> = {
               type: 'token',
               base: 'ethereum',
               network: slugify(name),
-              node_url: 'wss://mainnet.infura.io/ws',
+              node_url: 'wss://eth-mainnet.alchemyapi.io/v2/cNC4XfxR7biwO2bfIO5aKcs9EMPxTQfr',
```
[the remark this answers is not in the document]
Comment (jnaviask) [reply client/scripts/views/modals/create_community_modal.ts:756]: As I commented re Timothee, these hardcoded links are going to be removed entirely in the next PR.

**Pull request merged**